auth/postgres/init.go

```diff
@@ -65,6 +65,29 @@ func Migration() *migrate.MemoryMigrationSource {
                     `,
 				},
 			},
+			{
+				Id: "auth_4",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS pats (
+						id 					VARCHAR(36) PRIMARY KEY,
+                        name        		VARCHAR(254),
+						user_id	 			VARCHAR(36),
+						description			TEXT,
+						secret				TEXT,
+						issued_at			TIMESTAMP,
+						expires_at 			TIMESTAMP,
+						updated_at 			TIMESTAMP,
+						revoked 			BOOLEAN,
+						revoked_at 			TIMESTAMP,
+						entity_type 		TEXT,
+						last_used_at		TIMESTAMP,
```

Comment:
@nyagamunene 
Instead of having scope as text, can we have another table where pats respective scopes will be stored